x-pack/plugins/observability/public/config/paths.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const paths = {
+  observability: {
+    alerts: '/app/observability/alerts',
+  },
+  management: {
+    rules: '/app/management/insightsAndAlerting/triggersActions/rules',
+    ruleDetails: (ruleId: string) =>
+      `/app/management/insightsAndAlerting/triggersActions/rule/${encodeURI(ruleId)}`,
+    alertDetails: (alertId: string) =>
+      `/app/management/insightsAndAlerting/triggersActions/alert/${encodeURI(alertId)}`,
```

Comment:
Nit: note that `encodeURI` applies to whole URIs and does not encode the slash character `'/'` and [other special URI characters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI#encodeuri_vs_encodeuricomponent) which may result in routing errors. `encodeURIComponent` is safer for path components.